anyway/flask_app.py

```diff
@@ -1461,6 +1461,41 @@ def load_user(id: str) -> Users:
     return db.session.query(Users).get(id)
 
 
+# TODO: in the future add pagination if needed
+@app.route("/admin/get_all_users_info")
+def get_all_users_info() -> Response:
+    if current_user.is_anonymous:
+        return return_json_error(Es.BR_USER_NOT_LOGGED_IN)
+
+    admin_permission = Permission(RoleNeed(BE_CONST.Roles2Names.Admins.value))
+    if not admin_permission.can():
+        return return_json_error(Es.BR_MISSING_PERMISSION, BE_CONST.Roles2Names.Admins.value)
+
+    dict_ret = []
+    for user_obj in db.session.query(Users).all():
```

Comment:
It probably makes sense to sort the user by something, name, or date, or something